packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel.StubLibrary/src/StubLibraryVisitor.cs

```diff
@@ -46,6 +46,7 @@ internal class StubLibraryVisitor : ScmLibraryVisitor
         {
             if (!IsCallingBaseCtor(constructor) &&
                 !IsEffectivelyPublic(constructor.Signature.Modifiers) &&
+                !IsParameterlessInternalCtor(constructor) &&
```

Comment:
Why do we want to keep internal parameterless ctor?

Reply:
The SG will access it so it needs to remain.